kitsune/users/migrations/0009_Change_Twitter_link_to_Twitter_handle.py

```diff
@@ -21,8 +21,9 @@ def forwards(self, orm):
 
 
     def backwards(self, orm):
-        """Add twitter.com/ URL before the the Twitter username"""
-        for profile in orm.Profile.objects.exclude(twitter=None):
+        """Add twitter.com/ URL before the the Twitter username."""
+        # Backwards is different because this should not add link to the Null field and the field where there is no entry
```

Comment:
This line is too long. The linter won't catch it because the linter doesn't lint migrations, but you should wrap this to be no longer than 99 characters. In particular, it is bad when a line is too long for the GitHub UI.